docs/extensions/logging-and-monitoring.md

```diff
@@ -1,32 +1,39 @@
-# Logging and Monitoring configuration
+# Logging and Monitoring for Extensions
 
-By default, Gardener deploys a central Prometheus, AlertManager, and Grafana instance into the `garden` namespace of all seed clusters.
-Additionally, as part of the shoot reconciliation flow, it deploys a shoot-specific Prometheus, Grafana (and, if configured, an AlertManager) into the shoot namespace next to the other control plane components.
+Gardener provides an integrated logging and monitoring stack for alerting, monitoring and troubleshooting of its managed components by operators or end users. For further information how to make use of it in these roles, refer to the corresponding guides for [exploring logs](link) and [monitoring](link).
 
-Configurable by the `Logging` feature gate in the `gardenlet` configuration it might also deploy a central fluentd/fluent-bit, ElasticSearch, and Kibana deployment into the `garden` namespace of all seed clusters.
-Additionally, as part of the shoot reconciliation flow, it might deploy a shoot-specific ElasticSearch and Kibana into the shoot namespace next to the other control plane components.
+The components that constitute the logging and monitoring stack are managed by Gardener. By default, it deploys [Prometheus](https://prometheus.io/), [AlertManager](https://prometheus.io/docs/alerting/latest/alertmanager/) and [Grafana](https://grafana.com/) into the `garden` namespace of all seed clusters. If the `Logging` [feature gate](https://github.com/gardener/gardener/blob/master/docs/deployment/feature_gates.md) in the `gardenlet` configuration is enabled, it will deploy also [fluent-bit](https://fluentbit.io/) and [Loki](https://grafana.com/oss/loki/) there too.
+
+As part of the shoot reconciliation flow, Gardener deploys a shoot-specific Prometheus, Grafana and, if configured, an AlertManager into the shoot namespace, next to the other control plane components. If the `Logging` feature gate is enabled it deploys a shoot-specific Loki there too.
```

Comment:
~Duplicate of line 5?

Reply:
These are shoot-specific monitoring stack components (in shoots ns) vs seed global (in `garden` ns), i.e. the two paragraphs together describe the topology as a whole. 

Perhaps a diagram would be helpful here. Or alternatively, we should come up with a dedicated architecture document for the logging and monitoring that we can link to, because we come back to the topic in almost every document and it feels redundant.

Reply:
I rephrased to make ti clearer